### PR TITLE
modified Personal Data, Privacy, Security and Ethics Management

### PR DESCRIPTION
**Pull request by armandcismaru**

### CODE REVIEW

File: docs/portfolio_b.md [hunk 1]
```diff
@@ -128,24 +128,22 @@ We can then use these requirements to test our developed software solution.
...
+There will be no outbound communication from inside the data lake and the only inbound connection is data ingestion from the client API. All curated data marts are accessible from the lake S3 bucket via VPC or ACL with the Data Analytics team who are responsible for the security of their environment in which they deploy their processes. All access to these will be logged and anomalies, violations and exceptions will be flagged for alerts and auditing using AWS Cloudwatch. Furthermore, this storage of curated data marts is in a private network separate from the rest of the data lake in any case. A comprehensive look at this architecture is included in the Architecture section this paper.
 
 ### Ethics
 
-Ethics pre-approval was applied for on 19 November 2020, 12:00 GMT. The data handled in the project is simulated patients data supplied by the Healthcare Data Simulation project. The Healthcare Lake project does not collect any data from any real person nor handles any consent, as the scope of the project is to create a secure infrastructure for storing patients data.
```
Comment (jagosherred) [inline docs/portfolio_b.md:148]: I think we need to keep this in as its required to say we have ethics approval.

**Review by jagosherred — COMMENTED** [review]
looks good but reckon we need to keep the ethics pre-approval bit in

**Review by jagosherred — APPROVED** [review]
yeah nice reckon thats all good to merge

**Review by joekendal — APPROVED** [review]
Could use the terms encryption-at-rest and encryption-in-transit

**Pull request merged**